cfpb/consumerfinance.gov
# Docker postgreSQL version 13 upgrade

Try to bump the version of our PostgreSQL container from 10 to 11, 12, then 13, and check for any issues in loading a database dump, running our Cypress tests. 

The desired outcome of this PR: does anything break when we upgrade PostgreSQL?
The point is not to fix anything that does, but simply to discover if something does.

## Changes

- postgresql10 -> postgresql13 in Dockerfile
- Modified image: postgres:10-alpine -> postgres:13-alpine in docker-compose.yml and docker-stack.yml


## How to test this PR

Verify postgreSQL using these steps and commands:
1. docker stop $(docker ps -a -q)
1. docker-compose down
1. docker volume rm $(docker volume ls -q)
1. docker-compose build
1. docker-compose run
1. docker exec -it consumerfinancegov_python_1 bash
1. python cfgov/manage.py dbshell
1. python cfgov/manage.py migrate
1. ./refresh-data.sh production_django.sql.gz
1. python cfgov/manage.py search_index --rebuild
1. python cfgov/manage.py test
1. docker-compose -f docker-compose.e2e.yml run e2e


## Screenshots


## Notes and todos

- DATABASE_URL **must** contain a password
- `postgres://USER:PASSWORD@HOST:PORT/NAME`
- e.g. DATABASE_URL=postgres://cfpb:cfpb@localhost/cfgov
- If no obvious issues crop up, we should plan to upgrade our PostgreSQL RDS instance (open a CR, etc)
- If issues do crop up, we should add an epic to our roadmap to address them there before planning to update RDS


## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
